module2/homework/addEven/addEven.hpp

```diff
@@ -1,8 +1,13 @@
 #pragma once
 #include <vector>
 
-int addEven(const std::vector<int>& numbers) {
-    // TODO: Your implementation goes here
-    // Below return is only to make this function compile now
-    return -1;
+int addEven(const std::vector<int>& numbers) 
+{
+    int sum{};
+    for (const auto& el: numbers)
+    {
+        if(el%2==0)
+        sum=sum+el;
```

Comment:
```suggestion
        if(el % 2 ==0)
            sum = sum+el;
```
Use auto-formatting by clang-format

Reply:
and use curly braces even with one code line